src/io/flutter/logging/FlutterLogView.java

```diff
@@ -285,24 +282,7 @@ public void setSelected(AnActionEvent e, boolean state) {
 
     @Override
     public void actionPerformed(AnActionEvent e) {
-      ApplicationManager.getApplication().invokeLater(() -> {
```

Comment:
I refactoring by move this code to the `FlutterLogTree`.
So we can call `logTree.sendToClipboardSelectedItems()`

Reply:
Good idea!